custom_components/windcentrale/sensor.py

```diff
@@ -135,7 +136,7 @@ def extra_state_attributes(self):
             attr["Degrees"] = self.degrees.get(self._state)
         elif self.type == "windspeed" or self.type == "powertotal" or self.type == "powerpershare" or self.type == "powerpercentage" or self.type == "rpm":
             attr[CONF_STATE_CLASS] = SensorStateClass.MEASUREMENT
-        elif self.type == "energy" or self.type == "energyshares":
+        elif self.type == "energy" or self.type == "energyshares" or self.type == "yearproducedpercentage":
```

Comment:
This has no added value. This is only used for energy management tab

Reply:
Agreed... was testing something, which didn't work out.